.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
this folder should be added to .gitigonore

Reply:
done